package.json

```
@@ -22,7 +22,7 @@
   "bugs": "https://github.com/eslint-community/eslint-plugin-promise/issues",
   "scripts": {
     "format": "prettier --write .",
-    "lint": "eslint --report-unused-disable-directives .",
+    "lint": "eslint --report-unused-disable-directives . && prettier -c .",
```

Let's make it more clear what the `-c` flag stands for

```suggestion
    "lint": "eslint --report-unused-disable-directives . && prettier --check .",
```